torch/testing/__init__.py

```diff
@@ -96,6 +96,9 @@ def _compare_tensors_internal(a: torch.Tensor, b: torch.Tensor, *, rtol, atol, e
     # Compares complex tensors' real and imaginary parts separately.
     # (see NOTE Test Framework Tensor "Equality")
     if a.is_complex():
+        a = a.resolve_conj()
+        b = b.resolve_conj()
```

Comment:
just a thought: when we modify eq and ne -->
we could check conj bit for both tensors before resolving them. We only need to resolve the tensors if only one of the two has their conj bit set.

Reply:
Yes, you could write a custom Conj kernel for these, I think that's very reasonable.